### PR TITLE
feat(ingest): implement shortcuts for ingest

### PR DESCRIPTION
**Pull request by petrjasek**
- [x] toggle list/grid view with `t`
- [x] open search with `s`
- [x] fetch from ingest with `f`

### CODE REVIEW

File: client/app/scripts/superdesk-search/search.js [hunk 1]
```diff
@@ -569,7 +581,7 @@
                 scope: {
                     item: '='
                 },
-                template: '{{item.container}}',
+                Stemplate: '{{item.container}}',
```
Comment (amagdas) [inline client/app/scripts/superdesk-search/search.js]: Typo?

Comment (petrjasek) [inline client/app/scripts/superdesk-search/search.js]: :+1:

**Comment by amagdas** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**